winpr/include/winpr/wtsapi.h

```diff
@@ -116,7 +116,7 @@ typedef struct tagCHANNEL_PDU_HEADER
 #define VCEXPORT  __export
 #endif
 
-typedef VOID VCAPITYPE CHANNEL_INIT_EVENT_FN(LPVOID pInitHandle,
+typedef UINT VCAPITYPE CHANNEL_INIT_EVENT_FN(LPVOID pInitHandle,
```

Comment:
Changing this from VOID to UINT breaks compatibility with the Microsoft API: https://msdn.microsoft.com/en-us/library/windows/desktop/aa383568(v=vs.85).aspx
Not sure if we are still aiming for 100% MS API compatibility ..?